tests/test-dirs/syntax-document/language-extensions.t/first-class-modules.ml

```diff
@@ -0,0 +1,2 @@
+let sort (type s) (module Set : Set.S with type elt = s) l =
+  Set.elements (List.fold_right Set.add l Set.empty)
```

Comment:
Did you forget to move this one to the run.t file ?

Reply:
I hadn't concluded with it's node tree so I hadn't moved it to the `run.t` file yet.